receiver/githubreceiver/internal/traces/README.md

```diff
@@ -0,0 +1,168 @@
+# GitHub Actions Receiver
+
+<!-- status autogenerated section -->
```

Comment:
This whole auto generated section can be deleted. 

I think most of the information in this readme should be in the readme at the root of the receiver, and if necessary, can link to this as an internal document after adjustement.